Development-Guidelines/test-driven-flow.md

```diff
@@ -78,21 +78,21 @@ So to add a service, which requires data from the server we should:
 - Define the url in umbraco.httpbackend.js
 
 ### ServerVariables
-There is a static servervariables file in /mocks which describes the urls used by the rest service, this is currently needed as we don't have this set as an angular service, and no real conventions for these urls yet. Longer-term it would be great to have a urlBuilder which could do
+There is a static servervariables file in /mocks which describes the urls used by the rest service, this is currently needed as we don't have this set as an Angular service, and no real conventions for these urls yet. Longer-term it would be great to have a urlBuilder which could do
 
 ```javascript
 urlService.url("contentTypes", "GetAllowedChildren");
 // would return /<umbracodir>/<apibaseDir>/contentyTypes/getAllowedChildren
 ```
 
-But for now, they are set in the servervariables file.
+But for now, they are set in the server variables file.
```

Comment:
I'm fairly certain this is actually a contept in Umbraco, so let's keep this as one word!
I think we should even capitalize it like "ServerVariables" 🤔